Assets/HoloToolkit/UI/Scripts/Keyboard.cs

```diff
@@ -15,82 +15,82 @@
 
 namespace HoloToolkit.UI.Keyboard
 {
-	/// <summary>
-	/// A simple general use keyboard that is ideal for AR/VR applications.
-	/// </summary>
-	/// 
-	/// NOTE: This keyboard will not automatically appear when you select an InputField in your
-	///       Canvas. In order for the keyboard to appear you must call Keyboard.Instance.PresentKeyboard(string).
-	///       To retrieve the input from the Keyboard, subscribe to the textEntered event. Note that
-	///       tapping 'Close' on the Keyboard will not fire the textEntered event. You must tap 'Enter' to
-	///       get the textEntered event.
-	public class Keyboard : Singleton<Keyboard>
-	{
+    /// <summary>
+    /// A simple general use keyboard that is ideal for AR/VR applications.
```

Comment:
Oh yeah, we wrote that before we branded everything MR.  Or now from the unity standpoint is it XR?